less/column-layout.less

```diff
@@ -91,14 +83,15 @@
 .column-layout-cover .close {
   position: absolute;
   left: (@column-width + 20px);
-  top: 31px;
+  top: -1000px;
```

Comment:
I want the close button to be 'visible' so that the animation runs as expected, but I don't it to be visible to the user at this point, this moves it off the page.